parbase/FairParRootFileIo.h

```diff
@@ -30,13 +30,33 @@ class FairParRootFile : public TNamed
     FairRtdbRun* getRun() { return run; }
     void readVersions(FairRtdbRun*);
 
-    Bool_t IsOpen() { return RootFile->IsOpen(); }
-    void cd() { RootFile->cd(); }
-    Bool_t IsWritable() { return RootFile->IsWritable(); }
+    Bool_t IsOpen() { return RootFile && (RootFile->IsOpen()); }
+    void cd()
+    {
+        if (RootFile)
```

Comment:
Can you please add braces for the if statement blocks. Unfortunately, clang-format does not have a check for it. Not using braces here is a recipe for bugs down the line.